<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPA</class>
 <widget class="QWidget" name="WizardPA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="wizardTittleLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>VoxOx gives you an electronic Personal Assistant who answers the phone 
in your name, screens callers, and directs your calls accordingly.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>518</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/divisionPA.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose the voice of your new personal assistant:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="cmbAssistantVoice">
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>176</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/divisionPA.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Your assistant will refer to you as:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="txtName"/>
   </item>
   <item row="5" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>176</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/divisionPA.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Your Gender:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cmbUserGender">
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>438</width>
       <height>7</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/divisionPA.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="7">
    <widget class="QLabel" name="lblTip">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Tip: All options can be changed later from the Settings Area. You can also customize 
 all options for individual callers or groups from  the Contact Manager.</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
